esmvalcore/cmor/fix.py

```diff
@@ -163,94 +195,96 @@ def fix_metadata(cubes,
         for fix in fixes:
             cube_list = fix.fix_metadata(cube_list)
 
-        cube = _get_single_cube(cube_list, short_name, project, dataset)
-        checker = _get_cmor_checker(frequency=frequency,
-                                    table=project,
-                                    mip=mip,
-                                    short_name=short_name,
-                                    check_level=check_level,
-                                    fail_on_error=False,
-                                    automatic_fixes=True)
+        dataset_str = f'{project}:{dataset}'
+        cube = _get_single_cube(cube_list, short_name, dataset_str=dataset_str)
```

Comment:
```suggestion
        cube = cube_list[0]
```
would be cleaner. You're sure that there is only a single cube because GenericFix.fix_metadata ensures that and that is always applied.

Reply:
Done in https://github.com/ESMValGroup/ESMValCore/pull/2157/commits/c5054d750a52c9951908beb64ead81128488dbbf